Tensile/TensileCreateLibrary.py

```diff
@@ -1181,9 +1277,8 @@ def splitExtraParameters(par):
 
   assignGlobalParameters(arguments)
 
-  libraryPath = os.path.join(outputPath, "library")
-  ensurePath(libraryPath)
-  manifestFile = os.path.join(libraryPath, "TensileManifest.txt")
+  manifestFile = Path(outputPath)/TENSILE_LIBRARY_DIR/TENSILE_MANIFEST_FILENAME
```

Comment:
Path / str = Path?

Reply:
Yup, to make OS-agnostic paths with **pathlib.Path** you just separate by `/` and it will manage accordingly.